ui/demo/package.json

```
@@ -13,28 +13,30 @@
     "test": "echo '🚫 Tests are not configured for demo app.'"
   },
   "resolutions": {
-    "glob-parent": "5.1.2"
+    "glob-parent": "^6.0.0",
+    "nanoid": "^3.1.31",
+    "node-forge": "^1.3.0"
   },
   "dependencies": {
     "@types/react": "17.0.14",
     "@types/react-dom": "17.0.9",
-    "@types/react-pdf": "5.0.5",
+    "@types/react-pdf": "5.7.2",
     "@types/react-router-dom": "5.1.8",
     "antd": "4.16.7",
-    "axios": "0.21.1",
+    "axios": "^0.21.4",
```

We try to pin specific version of libraries. That way we can be sure any bugs we find are consistent across everyone's env.
```suggestion
    "axios": "0.21.4",
```